webpack.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  /* eslint-disable no-path-concat */
+  /* eslint-disable prefer-template */
   entry: __dirname + '/client/src/index.jsx',
```

Comment:
Does this work ok without using Node's path module? I've only seen it done with path before.